README.md

```diff
@@ -45,13 +50,22 @@ When you use the [JavaScript Api](https://github.com/metalsmith/metalsmith#api)
 metalsmith.source('src')
 metalsmith.destination('build')
 
-metalsmith.use(require('contentful-metalsmith')({ 'access_token' : 'YOUR_CONTENTFUL_ACCESS_TOKEN' }))
+metalsmith.use(require('contentful-metalsmith')({
+	'access_token': 'YOUR_CONTENTFUL_ACCESS_TOKEN',
+	'entry_key': '_key',
+	'entry_extension': 'md',
+	'contentful': {
+		'content_type': '2wKn6yEnZewu2SCCkus4as'
```

Comment:
Cool. What would be the best way to configure fetching all data from Contentful for users who want to build a site with multiple `content_types`? Should that be an array then instead of a string?